xbmc/addons/InputStream.cpp

```diff
@@ -98,7 +98,7 @@ bool CInputStream::Supports(CFileItem &fileitem)
   CVariant addon = fileitem.GetProperty("inputstreamaddon");
```

Comment:
Cool, I'll keep my eyes open. originally I was using the property "Addon.ID" because it was used elsewhere.. then that got ripped out :)